artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -621,11 +622,23 @@ public void initChannel(Channel channel) throws Exception {
 
             if (sslEnabled && !useServlet) {
 
-               SSLEngine engine;
+               final SSLContextConfig sslContextConfig = SSLContextConfig.builder()
+                  .withKeystoreProvider(realKeyStoreProvider)
```

Comment:
Maybe this is a nit-pick, but I prefer not having the "with" on the method names here.

Reply:
I can change that, no problem.

Reply:
Done.